admin/src/components/Overlay.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="component-overlay">
+    <b-jumbotron class="bg-light p-4">
+      <template #header>{{ items.firstName }}´s {{ $t('overlay.confirm.title') }}</template>
```

Comment:
This is a problem. First, if in English, it should be `'`. Second, this does not exits in German. I suggest to remove the name here, to avoid this Problem.
```suggestion
      <template #header>{{ $t('overlay.confirm.title') }}</template>
```